components/Forms/AddFoundItem.js

```diff
@@ -22,6 +23,7 @@ export default function AddFoundItem() {
     return (
       <>
         <StyledForm onSubmit={handleClick}>
+          <BurgerMenu />
```

Comment:
You could add the menu to your header/layout component. Then you don't have to add it to every page?

Reply:
Very good idea! :)